internal/tests/cli/boundary/_helpers.bash

```diff
@@ -24,11 +24,18 @@ function strip_all() {
 function has_status_code() {
   local json=$1
   local code=$2
-  if [ echo "$json"|jq -c ".status_code == $code" ]; then
-    return 1
-  fi
+  echo "checking .status_code == $code in $json"
+  echo "$json" | jq -e ".status_code == $code"
```

Comment:
Wouldn't it be more correct to use `.status` here?

Reply:
This is moving the other way, from `status` to `status_code`

Reply:
Doh, thanks